src/UserManager.ts

```diff
@@ -578,7 +570,7 @@ export class UserManager {
     /**
      * Returns promise to trigger a silent request (via an iframe) to the end session endpoint.
      */
-    public async signoutSilent(args: SignoutSilentArgs = {}): Promise<void> {
+    public async signoutSilent(args: SignoutSilentArgs = { }): Promise<void> {
```

Comment:
there is still a space left, funny that eslint did not see that :-), would be nice if you can remove that as well

Reply:
fixed